### ISSUE TITLE
Add Codeowners file

### ISSUE THREAD

**Issue by rishiosaur** [Standard-Structure/Standard-Structure#30]
We should probably have a system for automatically requesting code reviews from certain people. According to the Github docs, this should live in the `.github/CODEOWNERS` file, according to (this spec)[https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners].

**Comment by Matt-Gleich** [comment]
I agree. This is something we should definitely do. Something to talk about on the call.

**Comment by rishiosaur** [comment]
I don't think we talked about this on the call, @Matt-Gleich. Should I just do a poll in the Slack?

**Comment by Matt-Gleich** [comment]
Yeah
